ethereum/chain.py

```diff
@@ -324,7 +325,7 @@ def get_chain(self, frm=None, to=2**63 - 1):
         if frm is None:
             frm = int(self.db.get('GENESIS_NUMBER')) + 1
         chain = []
-        for i in xrange(frm, to):
+        for i in range(frm, to):
```

Comment:
Same here with using `itertools.count`.  It's not quite as perfect a use case but it circumvents the py2/3 juggling.

It also jumps out at me that *if* `to` is sufficiently low that `self.get_blockhash_by_number` will always return a hash this function will return `None` which seems like weird behavior.